admin_manual/installation/source_installation.rst

```diff
@@ -38,86 +38,118 @@ Apache and MariaDB, using `the ownCloud .tar archive
-* PHP module intl (increases language translation performance and fixes sorting 
-  of non-ASCII characters)
-* PHP module mcrypt (increases file encryption performance)
-* PHP module openssl (required for accessing HTTPS resources)
+* pdo_mysql (MySQL/MariaDB)
+* pgsql (requires PostgreSQL >= 9.0)
+* sqlite (>= 3, usually not recommended for performance reasons)
 
-Required for specific apps:
+Required For Specific Apps
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* PHP module ldap (for LDAP integration)
-* PHP module ftp (for FTP storage / external user authentication)
-* PHP module imap (for external user authentication)
-* PHP module smbclient  (SMB/CIFS integration)
+* ftp (for FTP storage / external user authentication)
+* imap (for external user authentication)
+* ldap (for LDAP integration)
+* smbclient  (SMB/CIFS integration)
   
 .. note:: SMB/Windows Network Drive mounts require the PHP module smbclient version 0.8.0+; see
   :doc:`../configuration_files/external_storage/smb`.
 
-Recommended for specific apps (*optional*):
-
-* PHP module exif (for image rotation in pictures app)
-* PHP module gmp (for SFTP storage)
-
-For enhanced server performance (*optional*) select one of the following 
-memcaches:
-
-* PHP module apc
-* PHP module apcu
-* PHP module memcached
-* PHP module redis (>= 2.2.6+, required for Transactional File Locking)
+Optional
+^^^^^^^^
+
+========= =====================================================================
+Extension Reason
+========= =====================================================================
+bz2       Required for extraction of apps
+curl      Highly recommended, as some functionality, such as HTTP user 
+          authentication, depends on this
+fileinfo  Highly recommended, as it enhances file analysis performance
+intl      Increases language translation performance and fixes sorting of
+          non-ASCII characters
+mcrypt    Increases file encryption performance
+openssl   Required for accessing HTTPS resources
+========= =====================================================================
+
+Recommended
+^^^^^^^^^^^
+
+For Specific Apps
+~~~~~~~~~~~~~~~~~
+
+========= =====================================================================
+Extension Reason
+========= =====================================================================
+exif      For image rotation in pictures app
+gmp       For SFTP storage
+========= =====================================================================
+
+For Server Performance
+~~~~~~~~~~~~~~~~~~~~~~
+
+For enhanced server performance consider installing one of the following cache
+extensions:
+
+* apc
+* apcu
+* memcached
+* redis (>= 2.2.6+, required for transactional file locking)
 
 See :doc:`../configuration_server/caching_configuration` to learn how to select 
 and configure a memcache.
 
-For preview generation (*optional*):
+For Preview Generation
+~~~~~~~~~~~~~~~~~~~~~~
 
-* PHP module imagick
+* imagick
```

Comment:
since the others are executables, maybe `PHP extension imagick` is better ... or ad a note explaining that video / office previews require avconf / ffmpeg / *office?